app/Resources/translations/messages.sv.xliff

```diff
@@ -518,6 +518,31 @@
                     Best regards,
                     Kompisbyrån</target>
             </trans-unit>
+            <trans-unit id="match.email.user">
+                <source>match.email.user</source>
+                <target>Hello %user_name%!
+
+Great that you want to be part of Kompisbyrån!
```

Comment:
Ok I already did the translation. @rifeman2007  you can find it here:
https://github.com/kompisbyran/Kompisbyran/commit/342be7f4af52c9f571fbe2d8ec5a636d98de6a28